network/validator/pubsub/authorized_sender_validator.go

```diff
@@ -101,7 +101,7 @@ func getRoles(channel network.Channel, msgTypeCode uint8) (flow.RoleList, error)
 	}
 
 	// cluster channels have a dynamic channel name
-	if msgTypeCode == cborcodec.CodeClusterBlockProposal || msgTypeCode == cborcodec.CodeClusterBlockVote || msgTypeCode == cborcodec.CodeClusterBlockResponse {
+	if channels.IsClusterChannel(channel) {
```

Comment:
So we no longer check the message type for cluster channels? just wondering is it fine to send any arbitrary message on a cluster channel?

Reply:
Good catch https://github.com/onflow/flow-go/pull/2495/commits/6b9bcd9dec58f93706065cdf0a60172328754308